dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/tcp/handler/DssTcpSslChannelInitializer.java

```diff
@@ -0,0 +1,50 @@
+package io.github.ztkmkoo.dss.core.network.tcp.handler;
+
+import java.security.cert.CertificateException;
+import java.util.Objects;
+
+import javax.net.ssl.SSLException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.netty.channel.ChannelPipeline;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.handler.codec.bytes.ByteArrayDecoder;
+import io.netty.handler.codec.bytes.ByteArrayEncoder;
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import io.netty.handler.ssl.util.SelfSignedCertificate;
+
+public class DssTcpSslChannelInitializer extends DssTcpChannelInitializer {
+
+    private final Logger logger = LoggerFactory.getLogger(DssTcpSslChannelInitializer.class);
+
+    private final SslContext sslCtx;
+
+    public DssTcpSslChannelInitializer(SslContext sslContext) throws InterruptedException {
+        this.sslCtx = (Objects.nonNull(sslContext)) ? sslContext : selfSignedSslContext();
+    }
+
+    @Override
+    protected void initChannel(SocketChannel ch) {
+        logger.info("Try to initChannel");
+
+        final ChannelPipeline p = ch.pipeline();
+
+        p.addLast(sslCtx.newHandler(ch.alloc()));
+        p.addLast(new ByteArrayDecoder());
+        p.addLast(new ByteArrayEncoder());
+        p.addLast(new DssTcpHandler());
```

Comment:
Hi, ByteBuf type casting occur in `ChannelRead0()` of DssTcpHandler.java.

So, I think it would be better to remove ByteArrayCodec.
```java
// add only this handler
p.addLast(new DssTcpHandler()); 
```

Reply:
https://github.com/ztkmkoo/dss/pull/114/commits/cdf94393880b3a367df4a2caa9a162837934f508

I saw him removed other handler this morning.
Why it remained?

Reply:
there are two channelInitializers.
one with ssl and the other without ssl

I forgot to remove both. sorry